src/main/target/KAKUTEH7/config.c

```diff
@@ -0,0 +1,36 @@
+#include <stdint.h>
```

Comment:
After #10791 , all new H743 targets should stop submitting legacy targets and start to submit them in the unified targets.